readme.md

```diff
@@ -1,235 +1,369 @@
-# Epic Online Services Plugin for Unity
+<img src="docs/images/PlayEveryWareLogo.gif" alt="Lobby Screenshot" width="5%"/>
 
-## Overview
-The [eos_plugin_for_unity repository](https://github.com/PlayEveryWare/eos_plugin_for_unity) contains the source code for development and support of the [Epic Online Services Plugin for Unity (UPM Package)](https://github.com/PlayEveryWare/eos_plugin_for_unity_upm) package.
+<div align="center"> <img src="docs/images/EOSPluginImage.gif" alt="Epic Online Services Plugin for Unity" /> </div>
+<br /><br /><br />
 
-Things this plugin provides:
+---
+
+
+
+
+  Table of Contents
+  <ol>
+    <li>
+      <a href="#overview">Overview</a>
+      <ul>
+        <li><a href="#supported-platforms">Supported Platforms</a></li>
+        <li><a href="#supported-eos-sdk-features">Supported EOS SDK Features</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#importing-the-plugin">Importing the Plugin</a></li>
+        <li><a href="#samples">Samples</a></li>
+        <li><a href="#configuring-the-plugin">Configuring the Plugin</a></li>
+      </ul>
+    </li>
+    <li><a href="#plugin-support">Plugin Support</a></li>
+    <li><a href="#source-code-contributor-notes">Source Code Contributor Notes</a></li>
+    <li><a href="#faq">FAQ</a></li>
+  </ol>
+
+
+# <div align="center">$\textcolor{deeppink}{\textsf{Overview}}$</div> <a name="overview" />
+---
+
+The Epic Online Services (EOS) Plugin for Unity is a software development kit (SDK) used for integrating EOS into a Unity project. Bringing the access free services that connect players across all platforms and all stores, to Unity in an easy to use package. Find more information on EOS [here](https://dev.epicgames.com/en-US/services) and Epic docs [here](https://dev.epicgames.com/docs/epic-online-services).
```

Comment:
Is the phrasing "Bringing the access free services" correct?

Reply:
removed 'access'